modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Step.scala

```diff
@@ -23,31 +23,40 @@ final case class Step(
   config: StepConfig,
   resources: Set[Resource],
   breakpoint: Boolean,
-  skip: Boolean,
+  skipped: Boolean,
+  skipMark: Boolean,
```

Comment:
It's tedious, but I think worthwhile in the end to define types for each of these `Boolean`s.

Reply:
I agree

Reply:
I don't know much about the seqexec model so pardon my ignorance.  Can a `Step` be a breakpoint and yet skipped and a `skipMark`?  I'm just wondering if the model is permitting combinations that don't make sense or shouldn't occur in reality.

Reply:
About the first comment, I agree. I even had a bug because I confused the values.

Reply:
A step can be marked to be skipped an have a breakpoint, but when run, the skip overrides the breakpoint. Currently the UI allows to set them independently. As long as the step is not run, the user can set the skip mark on a step that has a breakpoint. If he/she then unsets the skip mark, the breakpoint should remain.